{'description': {'en': 'Gentoo cross-compilation overlay for embedded devices'},
 'feed': ['https://cgit.gentoo.org/proj/embedded-cross.git/atom/'],
 'homepage': 'https://cgit.gentoo.org/proj/embedded-cross.git/',
 'name': 'embedded-cross',
 'owner': [{'email': 'armin76@gentoo.org',
            'name': u'Ra\xfal Porcel',
            'type': 'project'}],
 'quality': 'experimental',
 'source': [{'type': 'git',
             'uri': 'git://anongit.gentoo.org/proj/embedded-cross.git'},
            {'type': 'git',
             'uri': 'https://anongit.gentoo.org/git/proj/embedded-cross.git'},
            {'type': 'git',
             'uri': 'git+ssh://git@git.gentoo.org/proj/embedded-cross.git'}],
 'status': 'official'}
$ pmaint sync embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
Already up-to-date.
*** syncing embedded-cross
*** synced embedded-cross
 * Sync succeeded
$ pmaint regen --use-local-desc --pkg-desc-index -t 2 embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.063 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.062 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.061 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.


!!! ERROR: app-editors/vim-core-7.2.303 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.


!!! ERROR: app-editors/vim-core-7.3 failed.
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'

!!! ERROR: x11-libs/gtk+-2.14.5 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '187' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'

!!! ERROR: app-editors/vim-core-7.0.235 failed.
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'

!!! ERROR: app-editors/vim-core-7.0.243 failed.
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '176' 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!        vim-core-7.2.303.ebuild, line   6:  Called inherit 'vim'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'

!!! ERROR: app-editors/vim-core-7.1.123 failed.

!!! ERROR: x11-drivers/xf86-video-glamo-20100626 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'

!!! ERROR: x11-drivers/xf86-video-glamo-9999 failed.
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'

!!! ERROR: app-editors/vim-core-7.2.442 failed.
!!!                 ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!            vim-core-7.3.ebuild, line   7:  Called inherit 'vim'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'

!!! ERROR: app-editors/vim-core-7.1.330 failed.
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
 * 
 * "/home/mgorny/repos/embedded-cross/sci-geosciences/gpsd/gpsd-2.37.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                 ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop

!!! ERROR: app-editors/vim-7.3 failed.
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                 ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '248' 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
!!!             gtk+-2.14.5.ebuild, line   7:  Called inherit 'gnome.org' 'flag-o-matic' 'eutils' 'libtool' 'virtualx'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '228' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'virtualx'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
!!!                 ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!        vim-core-7.0.243.ebuild, line   5:  Called inherit 'vim'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'

!!! ERROR: app-misc/mdbus-9999 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!                         ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!        vim-core-7.0.235.ebuild, line   5:  Called inherit 'vim'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '160' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'
!!!                         ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!                virtualx.eclass, line  38:  Called die
!!! The die message:
!!!  virtualx eclass require EAPI=2 or newer.

!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!             isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!        vim-core-7.1.123.ebuild, line   5:  Called inherit 'vim'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '165' 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'

!!! ERROR: x11-terms/vala-terminal-1.2 failed.
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!   xf86-video-glamo-9999.ebuild, line  10:  Called inherit 'x-modular'
!!!   xf86-video-glamo-20100626.ebuild, line  16:  Called inherit 'x-modular'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'

!!! ERROR: x11-wm/enlightenment-0.16.999.061-r1 failed.

!!! ERROR: dev-util/vala-dbus-binding-tool-0.1.6 failed.
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'

!!! ERROR: x11-wm/enlightenment-0.16.999.063 failed.

!!! ERROR: x11-terms/vala-terminal-1.1.1 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!        vim-core-7.1.330.ebuild, line   5:  Called inherit 'vim'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!                  ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!                 vim-7.3.ebuild, line   7:  Called inherit 'vim'
!!!                    ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!        vim-core-7.2.442.ebuild, line   6:  Called inherit 'vim'
!!!             isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!                  ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'

!!! ERROR: dev-lang/vala-0.8.1.1 failed.
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                    ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 188' for eclass vim

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
received unknown com: export CATEGORY='dev-lang' PR=r1 PVR='2.6.4-r1' PV='2.6.4' EBUILD='/home/mgorny/repos/embedded-cross/dev-lang/python/python-2.6.4-r1.ebuild' P='python-2.6.4' PF='python-2.6.4-r1' PN=pythonfailed
!!!                   x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
Exception in thread Thread-9:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '208' 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                  ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 178' for eclass vim

received unknown com: export CATEGORY='dev-lang' PR=r8 PVR='5.8.8-r8' PV='5.8.8' EBUILD='/home/mgorny/repos/embedded-cross/dev-lang/perl/perl-5.8.8-r8.ebuild' P='perl-5.8.8' PF='perl-5.8.8-r8' PN=perlfailed
Exception in thread Thread-15:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                    ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '261' 'depend'
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                  ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '245' 'depend'
!!!                  ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!              mdbus-9999.ebuild, line   5:  Called inherit 'git'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '218' 'depend'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!               x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!!                    ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '254' 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!                  ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'

!!! ERROR: dev-lang/python-2.6.2-r1 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                            ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-util/vala-dbus-binding-tool/vala-dbus-binding-tool-0.1.6.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '179' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.2.303
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.3
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.0.235
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.0.243
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.1.123
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-20100626
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-editors/vim-core-7.2.442
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: dev-lang/perl-5.8.8-r8
!!! caught exception Metadata Exception: pkg ebuild src: sci-geosciences/fso-gpsd-0.8, attr eapi
error: parsed eapi '0' doesn't match sourced eapi '2' while processing ebuild src: sci-geosciences/fso-gpsd-0.8
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.1.330
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-editors/vim-7.3
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: dev-lang/python-2.6.4-r1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: app-misc/mdbus-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-util/strace-4.5.15
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/gtk+-2.14.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-util/dialog-1.1.20080819
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.2
ERROR pkgcore: !!! caught exception Metadata Exception: pkg ebuild src: x11-wm/enlightenment-0.16.999.061-r1, attr eapi
error: parsed eapi '2' doesn't match sourced eapi '0' while processing ebuild src: x11-wm/enlightenment-0.16.999.061-r1
returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-util/vala-dbus-binding-tool-0.1.6
!!! caught exception Metadata Exception: pkg ebuild src: x11-wm/enlightenment-0.16.999.063, attr eapi
error: parsed eapi '2' doesn't match sourced eapi '0' while processing ebuild src: x11-wm/enlightenment-0.16.999.063
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.1.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-admin/metalog-0.8-r1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/freesmartphone-framework-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/python-2.6.2-r1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/perl-5.8.8-r9
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/fso-raw-1.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-libs/gnutls-2.8.5
ERROR pkgcore:!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'

!!! ERROR: net-wireless/wmiconfig-9999 failed.

!!! ERROR: sys-mobilephone/fsodatad-0.1.0 failed.

!!! ERROR: sys-mobilephone/fsodeviced-0.9.4 failed.
!!!                          ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.063.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                            ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-util/vala-dbus-binding-tool/vala-dbus-binding-tool-0.1.6.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
 * 
!!! Call stack:
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20091021.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
!!! Call stack:
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'

!!! ERROR: sys-mobilephone/freesmartphone-framework-20091021 failed.
!!!                             ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.061-r1.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! Call stack:
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                          ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.063.ebuild'
!!!                isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/dev-util/vala-dbus-binding-tool/vala-dbus-binding-tool-0.1.6.ebuild'

!!! ERROR: sys-mobilephone/fsomusicd-0.0.0 failed.
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                             ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.061-r1.ebuild'
!!!       vala-terminal-1.2.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '188' 'depend'
!!!                         ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!              isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.063.ebuild'
!!!                         ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!   vala-dbus-binding-tool-0.1.6.ebuild, line   6:  Called inherit 'base'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                 isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.061-r1.ebuild'

!!! ERROR: x11-base/xglamo-20081205-r2 failed.
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '203' 'depend'
!!!          ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild'
!!!                         ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '296' 'depend'
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!   enlightenment-0.16.999.063.ebuild, line   8:  Called inherit 'enlightenment'
!!!     vala-terminal-1.1.1.ebuild, line   5:  Called inherit 'git' 'autotools'

!!! ERROR: sys-mobilephone/freesmartphone-framework-20090704 failed.
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '193' 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-lang/python/python-2.6.2-r1.ebuild'

!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5.5 failed.
!!!   enlightenment-0.16.999.061-r1.ebuild, line   8:  Called inherit 'enlightenment'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!!                     ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 183' for eclass base

!!! Call stack:
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='2.0.0' PV='2.0.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild' P='apm2-2.0.0' PF='apm2-2.0.0' PN=apm2path
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                         ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!            vala-0.8.1.1.ebuild, line   7:  Called inherit 'gnome2'
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'

!!! ERROR: sys-mobilephone/freesmartphone-framework-0.5 failed.
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '213' 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!           ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-lang/python/python-2.6.2-r1.ebuild'
!!!                   ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/net-wireless/wmiconfig/wmiconfig-9999.ebuild'
!!!  unknown inherit command from pythonic side, 'gen_metadata 285' for eclass enlightenment
!!!                                 ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20091021.ebuild'

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.5.1' PV='0.5.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild' P='freesmartphone-framework-0.5.1' PF='freesmartphone-framework-0.5.1' PN='freesmartphone-framework'path
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                         ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '208' 'depend'
!!!                      ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 215' for eclass enlightenment

received unknown com: export CATEGORY='x11-base' PR=r6 PVR='1.5.3-r6' PV='1.5.3' EBUILD='/home/mgorny/repos/embedded-cross/x11-base/xorg-server/xorg-server-1.5.3-r6.ebuild' P='xorg-server-1.5.3' PF='xorg-server-1.5.3-r6' PN='xorg-server'path
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild'
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gnome2'
!!!                    ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                      ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/net-wireless/wmiconfig/wmiconfig-9999.ebuild'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/dev-lang/python/python-2.6.2-r1.ebuild'
!!!                         ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                                 ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20091021.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                    ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '201' 'depend'
!!!                      ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/net-wireless/wmiconfig/wmiconfig-9999.ebuild'
!!!                         ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '296' 'depend'
!!!         python-2.6.2-r1.ebuild, line  12:  Called inherit 'autotools' 'eutils' 'flag-o-matic' 'libtool' 'multilib' 'python' 'toolchain-funcs' 'versionator'
!!!                     isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20091021.ebuild'
!!!                    ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '275' 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!          wmiconfig-9999.ebuild, line   5:  Called inherit 'subversion' 'toolchain-funcs'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!                         ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                      ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '285' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'subversion'
!!!                  gnome2.eclass, line  19:  Called die
!!! The die message:
!!!  EAPI=1 is not supported

!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'python'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 198' for eclass subversion

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.5.0' PV='0.5.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild' P='fsogsmd-0.5.0' PF='fsogsmd-0.5.0' PN=fsogsmdpath
Exception in thread Thread-10:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   freesmartphone-framework-20091021.ebuild, line   6:  Called inherit 'git'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'
!!!          fsodatad-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                    ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                            ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!        fsodeviced-0.9.4.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                            ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild'
!!!                                 ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild'
!!!                      ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 205' for eclass python

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='1.0.0' PV='1.0.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild' P='fso-alsa-1.0.0' PF='fso-alsa-1.0.0' PN='fso-alsa'path
!!!               ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe


!!! ERROR: sys-mobilephone/libframeworkd-phonegui-20090724 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                              ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                          ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                                 ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild'

!!! ERROR: sys-mobilephone/fsotdld-0.3.0 failed.
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!!                isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.ebuild'
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 203' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.1.0' PV='0.1.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild' P='fsoinitd-0.1.0' PF='fsoinitd-0.1.0' PN=fsoinitdpath
Exception in thread Thread-14:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!      xglamo-20081205-r2.ebuild, line   5:  Called inherit 'autotools' 'git' 'toolchain-funcs'
!!!   freesmartphone-framework-0.5.ebuild, line   5:  Called inherit 'distutils'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'autotools'
!!!                     isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild'
!!!         fsomusicd-0.0.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 228' for eclass autotools

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.3.1' PV='0.9.3.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/gsm0710muxd/gsm0710muxd-0.9.3.1.ebuild' P='gsm0710muxd-0.9.3.1' PF='gsm0710muxd-0.9.3.1' PN=gsm0710muxdpath
Exception in thread Thread-5:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                              ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '198' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!                       ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!          ebuild-default-functions.lib, line 230:  Called __internal_inherit 'distutils'
!!!   freesmartphone-framework-20090704.ebuild, line   6:  Called inherit 'git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!                       ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 208' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.4' PV='0.9.4' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fsousaged/fsousaged-0.9.4.ebuild' P='fsousaged-0.9.4' PF='fsousaged-0.9.4' PN=fsousagedpath
!!!                  isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.5.ebuild'
Exception in thread Thread-16:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 306' for eclass distutils

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090506 PV=20090506 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090506.ebuild' P='libframeworkd-phonegui-efl-20090506' PF='libframeworkd-phonegui-efl-20090506' PN='libframeworkd-phonegui-efl'path
!!!                       ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '286' 'depend'
 returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-wireless/wmiconfig-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-libs/webkit-efl-0.1_p20090703
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fsodeviced-0.9.4
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-20091021
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-base/xglamo-20081205-r2
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-20090704
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/apm2-2.0.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-base/xorg-server-1.5.3-r6
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090506
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/vala-0.8.1.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsogsmd-0.5.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fso-alsa-1.0.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090724
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsotdld-0.3.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsoinitd-0.1.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libfsotransport-0.9.6
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/gsm0710muxd-0.9.3.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libgsm0710mux-0.9.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsousaged-0.9.4
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libphone-ui-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090506
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing Exception in thread Thread-8:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!               ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!          cornucopia_git.eclass, line   6:  Called inherit 'base' 'git' 'autotools'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!                          ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!           fsotdld-0.3.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!   freesmartphone-framework-0.5.5.ebuild, line   6:  Called inherit 'git'

!!! ERROR: sys-mobilephone/libframeworkd-glib-0.1 failed.
!!!                       ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 233' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.1.1' PV='0.1.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsoresource/libfsoresource-0.1.1.ebuild' P='libfsoresource-0.1.1' PF='libfsoresource-0.1.1' PN=libfsoresourcepath
Exception in thread Thread-12:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!              ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: sys-mobilephone/fsonetworkd-0.1.0 failed.
!!!            ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                               ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090724.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                       ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!              ebuild-daemon.bash, line 210:  Called __ebd_main_loop

!!! ERROR: sys-mobilephone/libframeworkd-glib-20090227 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                               ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090724.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!              ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '245' 'depend'
!!!                   ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '218' 'depend'
!!!                   ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                   isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090724.ebuild'
!!!              ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!   libframeworkd-phonegui-20090724.ebuild, line   5:  Called inherit 'shr'
!!!             ebuild-default-functions.lib, line 230:  Called __internal_inherit 'shr'
!!!                      ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!!                   ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '266' 'depend'
!!!                        ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/shr.eclass'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                   ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild'
!!!                   isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/eclass/shr.eclass'
!!!                           ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild'
!!!                           ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!               isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild'
!!!                               shr.eclass, line   5:  Called inherit 'autotools' 'eutils' 'git'
!!!   libframeworkd-glib-20090227.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!       fsonetworkd-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                      ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!!             ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                        ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!          isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild'
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'

!!! ERROR: sys-mobilephone/libfsoframework-0.8.1 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 224' for eclass git

!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 224' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090614 PV=20090614 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/ophonekitd/ophonekitd-20090614.ebuild' P='ophonekitd-20090614' PF='ophonekitd-20090614' PN=ophonekitdfailed
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090506 PV=20090506 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/ophonekitd/ophonekitd-20090506.ebuild' P='ophonekitd-20090506' PF='ophonekitd-20090506' PN=ophonekitdpath
Exception in thread Thread-7:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

Exception in thread Thread-13:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   libframeworkd-glib-0.1.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!    ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!               ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 193' for eclass git

!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '238' 'depend'
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.1.0' PV='0.1.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/mioctl/mioctl-0.1.0.ebuild' P='mioctl-0.1.0' PF='mioctl-0.1.0' PN=mioctlfailed
Exception in thread Thread-6:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsoframework/libfsoframework-0.8.1.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsoframework/libfsoframework-0.8.1.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsoframework/libfsoframework-0.8.1.ebuild'
!!!   libfsoframework-0.8.1.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 224' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/ophonekitd/ophonekitd-99999999.ebuild' P='ophonekitd-99999999' PF='ophonekitd-99999999' PN=ophonekitdpath
Exception in thread Thread-11:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

ebuild src: sys-mobilephone/libphone-utils-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-0.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/mioctl-0.1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfsoresource-0.1.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsonetworkd-0.1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090227
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/ophonekitd-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfsoframework-0.8.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/ophonekitd-20090724
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/ophonekitd-20090506
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/mkdump-1.0.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/phoneuid-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/ophonekitd-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/pyphonelog-99999999
Error running an operation:
 Unhandled exception in regen_cache
 * Cache regen failed with 246
